src/Features/CSharp/Portable/Completion/KeywordRecommenders/AssemblyKeywordRecommender.cs

```diff
@@ -17,7 +17,7 @@ public AssemblyKeywordRecommender()
 
         protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
         {
-            if (context.IsTypeAttributeContext(cancellationToken))
+            if (context.TargetToken.GetNextToken().Kind() == SyntaxKind.NamespaceKeyword || context.IsTypeAttributeContext(cancellationToken) )
```

Comment:
Hey @lillo42 .  This is close, but isn't really the right way to fix this issue.  For example, with teh above fix, we would now see 'assembly' show up here:

```c#
$$
namespace Goo { }
```

It also wouldn't show up on:

```c#
[$$
[assembly: Whatever]
namespace N {}
```

To fix this, we'll likely want something like `context.IsGlobalAttributeContext(...)` (if it doesn't already exist) that enumerates the appropriate cases and conditions where this should show up.  

Let me know if you need anymore help/info with this!